src/App.js

```diff
@@ -1,13 +1,28 @@
 import BookList from './views/Books/BookList';
+import BookDetail from './views/Books/BookDetail';
 import './App.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 function App() {
-  // TODO: Add routes to books & views
   return (
-    <main className="container">
-      <h1>Library Catalog</h1>
-      <BookList />
-    </main>
+    <BrowserRouter>
+      <main className="container">
+        <h1>Library Catalog</h1>
+        <Switch>
+          <Route exact path='/'>
+            <p>check out our sick-ass <a href='/books'>books</a></p>
+          </Route>
+          <Route exact path='/books/:id'>
+            <a href='../'>&lt;&lt;&lt; go back to the books</a>
+            <BookDetail />
+          </Route>
+          <Route path='/books'>
+            <a href='../'>&lt;&lt;&lt; go back to the homepage</a>
+            <BookList />
```

Comment:
This directs me back to the home page and not the catalog page. As a user I would need to get through an extra page to get to catalog.